.github/workflows/spotbugs.yml

```diff
@@ -9,10 +9,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - name: Set up JDK 1.8
+    - name: Set up JDK 11
```

Comment:
Maybe version could be dropped from the name? Then - with the future change to come, there will be no need to keep them in sync.

Reply:
I'm used to see the JDK version in the build steps because when doing matrix builds (running the build on different JDK) it helps quickly seeing what's running. But I agree that we could remove it as well, I suppose it's matter of preference

Reply:
With the matrix build, that is not the case here, the matrix element could be used to build names. Here the literal `1.8`/`11` is used (twice).

The matrix with one JDK would do as well.

Currently it's the only step in the job and there is no different JDK involved so far. Hence I proposed to consider dropping version from the name.

Reply:
I am ok with being more explicit on the label name.